Valid Years Change

RevatureGentry opened the pull request:
Modified how the valid-years endpoints returns the years. Did not previously address the case when a batch wraps => starts in 2019, ends in 2020

RevatureGentry committed:
Valid Years Change

Modified how the valid-years endpoints returns the years. Did not previously address the case when a batch wraps => starts in 2019, ends in 2020

Quinn-Donnelly commented on src/main/java/com/revature/batchservice/service/BatchService.java:
```diff
@@ -156,8 +156,10 @@ public BatchEntity findBatchById(Integer id) {
-		return years.distinct().sorted(Comparator.reverseOrder()).collect(Collectors.toList());
+		List<Integer> years = br.findAllEndDates();
+		years.addAll(br.findAllStartDates());
+		System.err.println(years);
```
Do we need to commit in the print?

RevatureGentry committed:
Removed unneeded code

Removed commented out and System.out.println statements from
`findBatchYears()` method in BatchService
